modules/external-api/src/main/java/org/opencastproject/external/endpoint/EventsEndpoint.java

```diff
@@ -1726,14 +1731,24 @@ public Response updateEventScheduling(@HeaderParam("Accept") String acceptHeader
       logger.debug("Client sent unparsable scheduling information for event {}: {}", id, scheduling);
       return RestUtil.R.badRequest("Unparsable scheduling information");
     }
-    Optional<Response> clientError = updateSchedulingInformation(parsedJson, id, requestedVersion);
+    Optional<Response> clientError = updateSchedulingInformation(parsedJson, id, requestedVersion, allowConflict);
     return clientError.orElse(Response.noContent().build());
   }
 
   private Optional<Response> updateSchedulingInformation(
```

Comment:
As this private method is called only once, I would rather remove it and adapt the caller to use the extended version with allowConflict set to false.

Reply:
👍 ok removing the extra function.